bindings/generate_bindings.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+import subprocess
+from pathlib import Path
+
+def uniffi_gen(executable: str, *args):
+    args = list(str(a) for a in args)
+    uniffi = [executable]
+    uniffi.extend(args)
+    print(f'Running {" ".join(uniffi)}')
+    subprocess.run(uniffi, check=True)
+
+def uniffi_gen_go(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen-go", udl, "-o", output_dir)
+
+
+def uniffi_gen_cs(udl: Path, output_dir: Path, config_file: Path):
+    uniffi_gen("uniffi-bindgen-cs", udl, "-o", output_dir, "-c", config_file)
+
+
+def uniffi_gen_swift(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen", "generate", udl, "--language", "swift", "-o", output_dir)
+
+
+def uniffi_gen_kotlin(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen", "generate", udl, "--language", "kotlin", "-o", output_dir)
+
+def uniffi_gen_python(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen", "generate", udl, "--language", "python", "-o", output_dir)
+
+def uniffi_gen_cpp(udl: Path, output_dir: Path):
+    uniffi_gen("uniffi-bindgen-cpp", udl, "-o", output_dir)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="Generate UniFFI bindings from the UDL file", exit_on_error=True)
+    parser.add_argument("-d", "--directory", help="Output directory. Default set to './dist/ffi'")
+    parser.add_argument("-u", "--udl", help="UDL file")
```

Comment:
I am unaware that there's a config file possible.. Could you shed some light on it? I am aware that there's UDL file which is source for bindings. What about the config file? Thanks in advance!

Reply:
For some reason when I tried to generate csharp bindings it complained about the cdylib_name not being set. The fix for that (as done in moose) was to add this uniffi.toml where I specify it.